release/_package.sh

```diff
@@ -109,6 +109,7 @@ function generate_release_properties_file {
 		echo "bundle.url=https://releases-cdn.liferay.com/${LIFERAY_RELEASE_PRODUCT_NAME}/${_PRODUCT_VERSION}/${bundle_file_name}"
 		echo "git.hash.liferay-docker=${_BUILDER_SHA}"
 		echo "git.hash.liferay-portal-ee=${_GIT_SHA}"
+		echo "git.tag=${_ARTIFACT_VERSION}"
```

Comment:
@nikki-pru, it'll work only for DXP releases. For Portal/GA, you should use `_PRODUCT_VERSION` instead.